apps/ejabberd/src/mongoose_riak.erl

```diff
@@ -30,9 +30,11 @@
 -export([update_type/3, update_type/4]).
 -export([fetch_type/2, fetch_type/3]).
 -export([list_keys/1]).
+-export([list_buckets/1]).
```

Comment:
It's a helper function I used while developing. I left it because I found it useful.